nifi-commons/nifi-write-ahead-log/src/test/java/org/wali/TestMinimalLockingWriteAheadLog.java

```diff
@@ -538,7 +538,7 @@ public void testRecoverFileThatHasTrailingNULBytesAndTruncation() throws IOExcep
     @Test
     public void testRecoverFileThatHasTrailingNULBytesNoTruncation() throws IOException {
         final int numPartitions = 5;
-        final Path path = Paths.get("target/testRecoverFileThatHasTrailingNULBytes");
+        final Path path = Paths.get("target/testRecoverFileThatHasTrailingNULBytesNoTruncation");
```

Comment:
This change doesn't look like it belongs?

Reply:
Yep, accidental inclusion.